dev/tools/controllerbuilder/pkg/commands/generatetypes/generatetypescommand.go

```diff
@@ -173,6 +173,10 @@ func RunGenerateCRD(ctx context.Context, o *GenerateCRDOptions) error {
 			return err
 		}
 
+		if err := typeGenerator.WriteVisitedMessages(); err != nil {
```

Comment:
I think we should remove the changes for this file. It's not necessarily a long-term fix (more of a hack to unblock for generation temporarily).

Reply:
Done